pom.xml

```diff
@@ -66,6 +69,31 @@
 			<version>${vertx-unit.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.vertx</groupId>
+			<artifactId>vertx-kafka-client</artifactId>
+			<version>${vertx-kafka-client.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
```

Comment:
I guess that this dependency isn't needed just for the tests part. It should be part of the next step of using the Vert.x Kafka client. Isn't it ?